dapps/marketplace/src/pages/listings/ListingCards.js

```diff
@@ -133,7 +133,7 @@ require('react-styl')(`
     cursor: pointer
 
     .main-pic
-      padding-top: 66.6%
+      padding-top: 75%
```

Comment:
I have to update #2630 to keep this same ratio for those CTA listing cards before it gets merged.